qt.lua

```diff
@@ -575,6 +579,58 @@ function premake.extensions.qt.getQRCDependencies(fcfg)
 
 end
 
+--
+-- Checks if a file is a ts file.
+--
+-- @param filename
+--		The file name to check.
+-- @return
+--		true if the file needs to be run through the lrelease tool, false if not.
+--
+function premake.extensions.qt.isTS(filename)
+	return path.hasextension(filename, { ".ts" })
+end
+
+--
+-- Adds the custom build for a ts file.
+--
+-- @param fcfg
+--		The config for a single file.
+-- @param cfg
+--		The config of the project ?
+--
+function premake.extensions.qt.addTSCustomBuildRule(fcfg, cfg)
+	local qt = premake.extensions.qt
+
+	-- get the project's location (to make paths relative to it)
+	local projectloc = fcfg.project.location
+
+	-- create the output file name
+	local qtqmgenerateddir = fcfg.qtqmgenerateddir or cfg.qtqmgenerateddir
+	local output = path.join(qtqmgenerateddir or fcfg.directory, fcfg.basename .. ".qm")
+
+	-- create the moc command
+	local command = '"' .. fcfg.config.qtbinpath .. '/lrelease" "' .. fcfg.relpath .. '"'
+	command = command .. ' -qm "' .. path.getrelative(projectloc, output) .. '"'
+
+	-- now create the arguments
+	local arguments = ""
+
+	-- if we have custom commands, add them
+	local qttsargs = iif(#fcfg.qttsargs ~= 0, fcfg.qttsargs, cfg.qttsargs)
+	if #qttsargs ~= 0 then
+		table.foreachi(qttsargs, function (arg)
+			arguments = arguments .. ' "' .. arg .. '"'
+		end)
+		command = command .. arguments
+	end
+
+	-- add the custom build rule
+	fcfg.buildmessage = "lreleas'ing " .. fcfg.name
```

Comment:
I'd replace by something like `"Running lrelease on "`.
(I should also change the other build messages like this too... it would solve the issue you mentioned somewhere, with the single quote causing problems with some generators and make things consistent. I'll probably do that after this PR gets merged)

Reply:
Changed.